### DISCUSSION TITLE
[Board]: heltec Wireless Stick Lite

### DISCUSSION THREAD

**Discussion by lordrak007** [meshtastic/firmware#3795] [Ideas]
### SOC

ESP32

### Lora IC

SX1276

### Product Link

https://resource.heltec.cn/download/Wireless_Stick_Lite

### Description

Hi, i have made varianf for older Heltec WSL. Now is officially supported only WSL v3.
I have tested WSL and works nicelly.

**Comment by caveman99** [comment]
Thanks for your suggestion. This device is deprecated already and it uses the older SX1276 chipset. I am hesitant to add the support to the official repository without someone willing to maintain it. Essentially all devices before Heltec V3 generation may still be in this repository but they're unsuppported. If a future firmware breaks them they will stay broken.

**Comment by lordrak007** [comment]
Hi i understand of your hesitate. It has standar ESP32 like several already supported devices, co why the are broken then older wireless stick and wireless stick duo stop working also. I understand that. I read many parts of mestashtics source code to try understand it and result is working WSL v2 (or 1).
Yes, i can build it myself every nebo update of meshtashtics but my goal is to be able update there devices by your web flasher and to be builded together with other variants.

**Comment by serialrf433** [comment]
Its supported since this year: https://github.com/meshtastic/firmware/pull/5808

The prebuild images are still missing in the .zip firmware download. I opened a issue about this here: https://github.com/meshtastic/firmware/issues/7708
